### PR TITLE
Add test for VerificationService.init()

### PR DESCRIPTION
**Pull request by kuzdogan**
Adds the tests for #1462

### CODE REVIEW

**Comment by kuzdogan** [comment]
Following the issue: https://github.com/nock/nock/issues/2762

**Comment by manuelwedler** [comment]
They fixed the issue of nock. The renovate PR also ran fine now. The new version is in staging

File: services/server/test/unit/VerificationService.spec.ts [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  afterEach(function () {
+    // Ensure that all nock interceptors have been used
+    nock.isDone();
+  });
+
+  it("should initialize compilers", async function () {
+    before(function () {
+      rimraf.sync(config.get("solcRepo"));
+      rimraf.sync(config.get("solJsonRepo"));
+    });
```
Comment (manuelwedler) [inline services/server/test/unit/VerificationService.spec.ts:25]: The `before` hook should not be called inside a `it` block. The functions should rather be called directly here without wrapping in the before hook.

**Review by manuelwedler — APPROVED** [review]
Very small fix to be made. I already approved, so you can make the fix and merge afterwards directly.

**Review by Romek-cloud — COMMENTED** [review]
Done

File: package-lock.json [hunk 2]
```diff
@@ -37656,7 +37656,7 @@
         "chai": "4.4.1",
         "hardhat": "2.22.6",
         "mocha": "10.7.0",
-        "nock": "14.0.0-beta.9",
+        "nock": "^14.0.0-beta.9",
```
Comment (Romek-cloud) [inline package-lock.json:37659]: Ok

Comment (Romek-cloud) [inline package-lock.json:37659]: Ok

File: package-lock.json [hunk 3]
```diff
@@ -37721,7 +37721,7 @@
         "hardhat": "2.22.6",
         "mocha": "10.7.0",
         "mochawesome": "7.1.3",
-        "nock": "14.0.0-beta.9",
+        "nock": "^14.0.0-beta.9",
```
Comment (Romek-cloud) [inline package-lock.json:37724]: Ok

File: package-lock.json (hunk 2 above)
Comment (Romek-cloud) [inline package-lock.json:37659]: Path

File: package-lock.json (hunk 3 above)
Comment (Romek-cloud) [inline package-lock.json:37724]: Path

**Pull request merged**